react/tic-tac-toe/src/app/index.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
-
+import { BrowserRouter as Router, Redirect, Route } from 'react-router-dom';
 import Game from './screens/Game';
+import LogIn from './screens/LogIn';
+import Matches from './screens/Game/components/Matches';
 
 import '../scss/application.scss';
 
-function App() {
-  return <Game />;
-}
+const App = () => (
+  <Router>
+    <div>
```

Comment:
Is this div really necessary? If you need a container (I don't think so bbut I might be wrong)  use `Fragment`

Reply:
Yes i need to wrap all the "Route" 's in 1 element , because Router can only take 1 child, if i didnt undertood wrong. I will change it to Fragment then

:white_check_mark: